### DISCUSSION TITLE
my answer for #3569

### DISCUSSION THREAD

**Discussion by univerioiln233** [termux/termux-packages#16740] [Ideas]
https://github.com/termux/termux-packages/issues/3569
you need : chmod +x

**Comment by sylirre** [comment]
Don't dig for old closed issues, especially if they are locked.

There is a variety of reasons why issue get closed, but most common are:
* Completed - answered/fixed.
* Inactive - the original poster doesn't answer for long time for whatever reason.
* Irrelevant - the issue is not related to the repository subject.

https://github.com/termux/termux-packages/issues/3569 is the last two cases.

**Comment by univerioiln233** [comment]
您好，您的电邮我已收到，我会尽快回复。祝一切顺利。
